pom.xml

```diff
@@ -425,17 +436,17 @@
 	<repositories>
 		<repository>
 			<id>lib</id>
-			<name>Legacy dependencies Repository</name>
-			<url>file:${project.basedir}/repo</url>
+			<name>Legacy Dependencies Repository</name>
+			<url>file:./repo</url>
```

Comment:
what was wrong with `${project.basedir}`?   It seems more robust and descriptive.

Reply:
I found that in some conditions it was creating the folder `${project.basedir}/`, thus I'm not able to reproduce it.

That's fine, I restore it and I also replace old `${basedir}` (deprecated)